recipes/subunit/all/conanfile.py

```diff
@@ -73,8 +73,15 @@ def build_requirements(self):
             self.win_bash = True
             if not self.conf.get("tools.microsoft.bash:path", check_type=str):
                 self.tool_requires("msys2/cci.latest")
-        if is_msvc(self):
+        if self.version == "1.4.0":
+            if is_msvc(self):
+                self.tool_requires("automake/1.16.5")
+        else:
+            # version >=1.4.4 is kept on GitHub without configuration built
+            self.tool_requires("libtool/2.4.7")
+            self.tool_requires("pkgconf/2.2.0")
```

Comment:
```suggestion
```
There is no `PkgConfDeps` generator used in the recipe. Is the `pkgconf` tool dependency really being used?

Reply:
`configure.ac` uses `PKG_CHECK_...` which fails if `pkgconf` is not installed: https://github.com/testing-cabal/subunit/blob/98aa1076f8aed666262022634ddccdf9e345b13f/configure.ac#L64-L66 
I don't know `autoconf` well to understand whether it can be fixed by another way.